src/app/components/AboutPage.tsx

```diff
@@ -11,7 +11,7 @@ export default function AboutPage() {
   const { isOpen, onOpen, onOpenChange } = useDisclosure();
 
   return (
-    <div className="flex flex-col min-h-screen">
```

Comment:
❓ Is there a reason we wouldn't make this change in `/src/app/components/Layout.tsx`?

Reply:
I thought it might affect the skiplink, but it probably won't it's just a bunch of divs. I'll close this PR and try how it sounds.